src/main/java/com/team/buddyya/feed/service/FeedService.java

```diff
@@ -0,0 +1,103 @@
+package com.team.buddyya.feed.service;
+
+import com.team.buddyya.auth.domain.StudentInfo;
+import com.team.buddyya.feed.domain.Category;
+import com.team.buddyya.feed.domain.Feed;
+import com.team.buddyya.feed.domain.FeedUserAction;
+import com.team.buddyya.feed.dto.request.FeedCreateRequest;
+import com.team.buddyya.feed.dto.request.FeedListRequest;
+import com.team.buddyya.feed.dto.request.FeedUpdateRequest;
+import com.team.buddyya.feed.dto.response.FeedListItemResponse;
+import com.team.buddyya.feed.dto.response.FeedListResponse;
+import com.team.buddyya.feed.dto.response.FeedResponse;
+import com.team.buddyya.feed.exception.FeedException;
+import com.team.buddyya.feed.exception.FeedExceptionType;
+import com.team.buddyya.feed.respository.FeedRepository;
+import com.team.buddyya.student.domain.Student;
+import com.team.buddyya.student.exception.StudentException;
+import com.team.buddyya.student.exception.StudentExceptionType;
+import com.team.buddyya.student.repository.StudentRepository;
+import jakarta.transaction.Transactional;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Sort;
+import org.springframework.stereotype.Service;
+
+@Service
+@Transactional
+@RequiredArgsConstructor
+public class FeedService {
+
+    private final LikeSevice likeSevice;
+    private final BookmarkService bookmarkService;
+    private final CategoryService categoryService;
+    private final FeedRepository feedRepository;
+    private final StudentRepository studentRepository;
+
+    public Feed findFeedById(Long feedId) {
+        return feedRepository.findById(feedId).orElseThrow(() -> new FeedException(FeedExceptionType.FEED_NOT_FOUND));
```

Comment:
P3: 메서드 체이닝이 길어지면 다음과 같이 구분해 주는 것도 괜찮을 것 같아요~
```suggestion
        return feedRepository.findById(feedId)
                             .orElseThrow(() -> new FeedException(FeedExceptionType.FEED_NOT_FOUND));
```

Reply:
네 컨벤션이 어긋난 부분이 있었네요.
수정하겠습니다!